src/js/components/DataTableColumns/DataTableColumns.js

```diff
@@ -170,6 +170,7 @@ export const DataTableColumns = ({ drop, options, ...rest }) => {
       })}
       kind="toolbar"
       icon={<Splits />}
+      tip="Manage Columns"
```

Comment:
I think we should consider doing this via `messages` to follow how we localize other messages.

Reply:
Maybe look at how `tip` is done with DataFilters.js

Reply:
Taking a look!